css/styles.css

```diff
@@ -0,0 +1,126 @@
+* { 
+  box-sizing: border-box; 
+}  
+
+h1{
+  color: #fdf65e;
+  background-color: #000;
+  padding: 8px 8px 8px 8px;
+  margin:0px;
+  font-size:40px;
+}
+
+nav{
+  
+  background-color: #000;
+  padding: 8px 8px 8px 8px;
+  /*float: right;*/
+  display:block;
+  text-align: right;
+  border-bottom: #fdf65e solid 4px ;
+}
+a{
+  color:#fff;
+  font-size:30px;
+  text-decoration: none;
+  margin-right: 5px;
+}
+.speciallink{
+  font-style: bold;
+  font-family: impact;
+}
+
+h2{
+  text-align: center;
+  font-style: italic;
+  font-size:30px;
+  background-color: #c6c6c6;
+  margin:0px;
+  padding: 12px 12px 12px 12px;
+}
+
+.taste{
+color:#999999;
+text-decoration: line-through;
+}
+
+.are{
+border-bottom: solid #fdf65e;
+}
+
+.background{
+  position:center center fixed;
+  width:100%;
+  max-height:600px;
+  position:relative;
+  float:left;
+}
+
+img{  
+  width:100%;
+  position:relative;
+  float:left;
+  background-size: 100% auto;
+}
+.meetyourmatch{
+background-color: rgba(255,255,255,.8);
+box-shadow: 10px 10px 5px #888888;
+float:right;
+position: relative;
+
+bottom:500px;
+width:35%;
+margin:10px 40px 10px 10px;
+padding: 30px 30px 30px 30px;
+height: auto;
+}
+
+.capbutton{
+  background-color:#fdf65e;
+}
+
+h3{
+  font-size: 24px;
+}
+
+h4{
+  font-size: 24px;
+}
+
+article{
+  border-right:solid #999999 1px;
+  padding:20px 20px 20px 20px;
+  display:flex;
+  flex-direction:column;
+  float:left;
+
+}
+
+.bottomboxes{
+  width:100%;
+  display: flex;
+  justify-content: space-around;
+  float:left;
```

Comment:
@akallbold this can be removed 🔨

Reply:
@rmjames  which line are you referring to? Removed float per your comment above!